webview/src/setup/components/dvc/CliUnavailable.tsx

```diff
@@ -1,17 +1,21 @@
 import React, { PropsWithChildren } from 'react'
+import { useSelector } from 'react-redux'
 import styles from './styles.module.scss'
 import { Button } from '../../../shared/components/button/Button'
 import { EmptyState } from '../../../shared/components/emptyState/EmptyState'
+import { SetupState } from '../../store'
 
 type CliUnavailableProps = {
   installDvc: () => void
-  pythonBinPath: string | undefined
   setupWorkspace: () => void
 }
 
 export const CliUnavailable: React.FC<
   PropsWithChildren<CliUnavailableProps>
-> = ({ installDvc, pythonBinPath, setupWorkspace, children }) => {
+> = ({ installDvc, setupWorkspace, children }) => {
```

Comment:
Aren't `installDvc` and `setupWorkspace` exported from `messages.ts`? They could be removed from props as well

Reply:
Good catch! Will remove these props.